docs/release-notes/8.8.asciidoc

```diff
@@ -5,7 +5,7 @@
 [[release-notes-8.8.0]]
 === 8.8.0
 
-To view a detailed summary of new features and enhancements we've documented this release, check out our {security-guide}/whats-new.html[8.8 release highlights].
+To view a detailed summary of new features and enhancements we've documented this release, check out our {security-guide}/whats-new.html[{minor-version} release highlights].
```

Comment:
This is a tricky statement. Changing the link text to an attribute doesn't quite solve the problem because it'll still be a disconnect between the sentence's first part always saying/implying 8.8 (because it's hardwritten into the file) and the second part dynamically saying whatever the latest version is.
```suggestion
To view a detailed summary of the latest features and enhancements, check out our {security-guide}/whats-new.html[release highlights].
```